### PR TITLE
4 packages from c-cube/qcheck at 0.9

### PR DESCRIPTION
**Pull request by c-cube**
This pull-request concerns:
-`qcheck.0.9`
-`qcheck-alcotest.0.9`
-`qcheck-core.0.9`
-`qcheck-ounit.0.9`

---
* Homepage: https://github.com/c-cube/qcheck/
* Source repo: git+https://github.com/c-cube/qcheck.git
* Bug tracker: https://github.com/c-cube/qcheck/issues

---
:camel: Pull-request generated by opam-publish v2.0.0~beta

### CODE REVIEW

**Comment by c-cube** [comment]
Ah, well, spurious error on travis :man_shrugging:

**Comment by mseri** [comment]
Alcotest errors are because you are pushing to the 2.0.0 branch with 1.2 version opam files. Today here should be the switch to opam2, maybe it’s worth to wait for that

**Comment by kit-ty-kate** [comment]
Could you take care of the camelus errors?

**Comment by c-cube** [comment]
@kit-ty-kate please, I'm tired of having to satisfy every whim of some linter. If opam accepts these files and the packages are good, can we please move forward? I fought ~~dune-release~~ opam-publish for 2 hours yesterday, and my patience is limited. I don't care about warnings.

**Comment by mseri** [comment]
The CI failures are due to `debian-unstable` being unable to get files from ocamlforge (ounit in this case, maybe a problem with security certificates?). We have seen this issue in Travis in the cohttp repository as well.

**Comment by c-cube** [comment]
:tada: thanks!

File: packages/qcheck-alcotest/qcheck-alcotest.0.9/opam [hunk 1]
```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "simon.cruanes.2007@m4x.org"
+authors: "Simon Cruanes <simon.cruanes.2007@m4x.orgr>"
```
Comment (pmetzger) [inline packages/qcheck-alcotest/qcheck-alcotest.0.9/opam]: should this be @m4x.org? Similar problem in the other opam files.

Comment (c-cube) [reply packages/qcheck-alcotest/qcheck-alcotest.0.9/opam]: well spotted -_-
I'll fix in my repo, that's fine anyway since the real address is just above…

**Pull request merged**